corehq/apps/accounting/migrations/0033_auto__add_field_invoice_is_hidden_to_ops.py

```diff
@@ -0,0 +1,250 @@
+# encoding: utf-8
```

Comment:
sorry, I already added migration 0033. You can remove this migration, rebase on master, and then re-create the migrations to get 0035.